test/suites/dev/moonbase/test-staking/test-staking-locks6.ts

```diff
@@ -1,11 +1,11 @@
 import "@moonbeam-network/api-augment";
 import { beforeAll, describeSuite, expect } from "@moonwall/cli";
 import { GLMR, MIN_GLMR_DELEGATOR, alith, generateKeyringPair } from "@moonwall/util";
-import { fromBytes } from "viem";
+import { getDelegatorStakingFreeze } from "../../../../helpers";
 
 describeSuite({
-  id: "D023481",
-  title: "Staking - Locks - schedule revoke",
+  id: "D013479",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.